src/components/radio-button-group/RadioButtonGroup.tsx

```diff
@@ -62,16 +62,16 @@ export const RadioButtonGroup = forwardRef<HTMLDivElement, PropsWithLabels<Props
   return (
     <RadioGroupRoot
       {...radixRadioGroupProps}
-      aria-label={ariaLabel}
-      aria-labelledby={ariaLabelledBy}
+      {...(ariaLabel ? { 'aria-label': ariaLabel } : {})}
+      {...(ariaLabelledBy ? { 'aria-labelledby': ariaLabelledBy } : {})}
```

Comment:
I don't think that was needed since those are props we receive, they are either undefined or specified, no need to add extra logic around this.